sysz

```diff
@@ -265,6 +279,10 @@ while [[ -n $1 ]]; do
     MANAGERS=(system)
     shift
     ;;
+  -a | --all)
```

Comment:
I really don't want a flag for this because I don't like to have to think ahead before pulling calling sysz. Perhaps I want to see the contents of a template before restart an instance of it. I don't want to have to restart sysz to see the template. I'd prefer to make the startup time faster by streaming in the results and have the templates at the bottom of the list.